README.md

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
- [ ] In your README, kindly update your screenshot gif. You replaced the gif, but forgot to update the link location.

  ![boursen-scrnsht](https://user-images.githubusercontent.com/36542986/158594578-91246f85-9f4e-47eb-bbd9-144544a8b93f.jpg)